server/src/main/java/com/cloud/user/AccountManagerImpl.java

```diff
@@ -1687,6 +1692,7 @@ public AccountVO disableAccount(String accountName, Long domainId, Long accountI
     public AccountVO updateAccount(UpdateAccountCmd cmd) {
         Long accountId = cmd.getId();
         Long domainId = cmd.getDomainId();
+        Long roleId = cmd.getRoleId();
         String accountName = cmd.getAccountName();
         String newAccountName = cmd.getNewName();
         String networkDomain = cmd.getNetworkDomain();
```

Comment:
@bwsw while we are here, I have just realized that 'cmd.getDetails' might throw an NPE. Could you please add an NPE check for this call?